src/Compilers/CSharp/Portable/CSharpExtensions.cs

```diff
@@ -1619,6 +1619,17 @@ public static Conversion ClassifyConversion(this SemanticModel? semanticModel, i
             var csmodel = semanticModel as CSharpSemanticModel;
             return csmodel?.GetDeclaredSymbol(node, cancellationToken);
         }
+
+        /// <summary>
+        /// Given a local function declaration syntax, get the corresponding symbol.
+        /// </summary>
+#pragma warning disable RS0026
+        public static IMethodSymbol? GetDeclaredSymbol(this SemanticModel? semanticModel, LocalFunctionStatementSyntax node, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
>GetDeclaredSymbol [](http://example.com/codeflow?start=37&length=17)

We probably should have a unit-test that verifies that ```GetDeclaredSymbol``` called with ```LocalFunctionStatementSyntax``` returns ```IMethodSymbol``` statically.

Reply:
Something like:
```
IMethodSymbol m = model.GetDeclaredSymbol(...);
```
And verify that the symbol is what we expect.